plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -671,42 +666,8 @@ function ListMenuItem:update()
             if self.do_hint_opened and DocSettings:hasSidecarFile(self.filepath) then
                 self.been_opened = true
             end
-            -- No right widget by default, except in History
-            local wright
-            local wright_width = 0
-            local wright_right_padding = 0
-            if self.mandatory_func then
-                -- Currently only provided by History, giving the last time read.
-                -- If we have it, we need to build a more complex widget with
-                -- this date on the right
```

Comment:
What are we losing by removing all this?
The last read date of deleted files ?

Reply:
The last read date of files that have been deleted and if no sql bookinfo found.

In the meantime the last read date is kept in history.lua, so it can be displayed, this requires some cnahges to the ListMneu logic.

Reply:
What's the (technical only?) reason for removing these chunks of code?

Reply:
There is no more `mandatory_func` in History.

Reply:
It looks like I added mandatory_func just for that case? And your compulsive cleaning habits removed it?

Reply:
You added `mandatory_func` in https://github.com/koreader/koreader/pull/6593 to show the last read date.

Reply:
`mandatory_func` is not needed in History after https://github.com/koreader/koreader/pull/10156.